modules/oe_theme_content_entity_organisation/oe_theme_content_entity_organisation.install

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Module install file.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Config\FileStorage;
+use Drupal\Core\Entity\Entity\EntityViewDisplay;
+
+/**
+ * Implements hook_install().
+ */
+function oe_theme_content_entity_organisation_install() {
+  // If we are installing from config, we bail out.
+  if (\Drupal::isConfigSyncing() === TRUE) {
+    return;
+  }
+
+  // Override entity view displays.
```

Comment:
@nagyad I've provided a little bit another solution than you asked. This code can work with any types of configurations, not only with entity view display configurations.